src/utils.tsx

```diff
@@ -165,3 +166,33 @@ export const getFilterValuesFromExpression = (exp: FilterExpression | null): str
 
   return [];
 };
+
+export const getStateFromSessionStorage = (quizStateKey: string): QuizLocalReducerState | null => {
+  const state = window?.sessionStorage?.getItem(quizStateKey);
+
+  if (state) {
+    return JSON.parse(state);
+  }
+  return null;
+};
+
+export const resetQuizSessionStorageState = (quizStateKey: string) => () => {
+  window?.sessionStorage?.removeItem(quizStateKey);
+};
+
+export const logger = (action: any) => {
+  console.group(
```

Comment:
let's tell eslint that we are doin this on purpose, so it knows not to warn us about it :)
<img width="726" alt="Screenshot 2023-06-26 at 1 44 16 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/20070114/032be050-4674-4b0f-875c-3b0ce6e482f8">